src/home_robot_hw/home_robot_hw/stretch_client/modules/nav.py

```diff
@@ -0,0 +1,141 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Iterable
+
+import rospy
+from geometry_msgs.msg import Twist
+from std_srvs.srv import SetBoolRequest, TriggerRequest
+
+from home_robot.utils.geometry import sophus2xyt, xyt2sophus, xyt_base_to_global
+from home_robot_hw.constants import T_LOC_STABILIZE
+from home_robot_hw.ros.utils import matrix_to_pose_msg
+
+from .abstract import AbstractControlModule, enforce_enabled
+
+
+class StretchNavigationInterface(AbstractControlModule):
```

Comment:
StretchBaseControl?

Reply:
I thought about this but the manipulation control interface also commands the base.
I wanted to decouple hardware components from types of commands, thus the nav / manip split.

Reply:
fair enough! i do think i prefer "client" here but in the intersts of finishing this PR i think we can go with whatever is consistent with manipulation interface